cajita/unit_test/tstHypreStructuredSolver2d.hpp

```diff
@@ -196,6 +199,9 @@ poissonTest( const std::string& solver_type, const std::string& precond_type,
         for ( int j = owned_space.min( Dim::J ); j < owned_space.max( Dim::J );
               ++j )
             EXPECT_FLOAT_EQ( lhs_host( i, j, 0 ), lhs_ref_host( i, j, 0 ) );
+
+    // finalize hypre
+    HYPRE_Finalize();
```

Comment:
Does the solver need to be out of scope before HYPRE is finalized?

Reply:
@lebuller can you point to the docs where they discuss this kind of thing?